File: java_api/src/test/java/GeneralTest.java
import globalincidents.controller.GetIncidentsController;
import junit.framework.TestCase;
import org.json.JSONArray;
import org.springframework.util.LinkedMultiValueMap;
import org.springframework.util.MultiValueMap;

public class GeneralTest extends TestCase {

  public void testGetIncidents(){
    MultiValueMap<String, String> params = new LinkedMultiValueMap<>();
    params.add("limit", "1");

    GetIncidentsController incidents = new GetIncidentsController();
    incidents.getParams(params);
    String response = incidents.run(params);

    JSONArray json = new JSONArray(response);
    assertTrue(json.length() > 0);
  }
}

File: kotlin_api/src/test/kotlin/GeneralTest.kt
import globalincidents.controller.GetIncidentsController
import junit.framework.TestCase
import org.json.JSONArray
import org.springframework.util.LinkedMultiValueMap

class GeneralTest : TestCase() {

    fun testGetIncidents() {
        val params = LinkedMultiValueMap<String, String>()
        params.add("limit", "1")

        val incidents = GetIncidentsController()
        incidents.getParams(params)
        var response: String = incidents.run(params)

        val json = JSONArray(response)
        TestCase.assertTrue(json.length() > 0)
    }
}